Form-Controls/index.html

```diff
@@ -18,21 +18,20 @@ <h1>Product Pick</h1>
          <br>
          <br>
          <label for="email">Email</label>
-         <input type="email" id="email" name="name" required>
+         <input type="text" id="email" name="name" required>
```

Comment:
I believe you have changed the type from "text" to "email" and back to "text"? In order to validate an email, the type should be 'email'? Can you change it and test it?

Reply:
Nice, the email field validation works well now! 😃